### PR TITLE
Support truncate table xxx.

### LINKED ISSUE #114: Support MySQL "truncate table xxx"
**What would you like to be added**:

**Why is this needed**:

Support MySQL "truncate table xxx"

### PR DESCRIPTION
**Pull request by dongzl**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #114 .

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/runtime/ast/truncate.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ast
+
+import (
+	"github.com/pkg/errors"
```
Comment (AlexStocks) [inline pkg/runtime/ast/truncate.go:21]: split

File: pkg/runtime/plan/truncate.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package plan
+
+import (
+	"context"
+	"github.com/arana-db/arana/pkg/mysql"
```
Comment (AlexStocks) [inline pkg/runtime/plan/truncate.go:22]: split

**Review by jjeffcaii — APPROVED** [review]
👍

**Review by Lvnszn — APPROVED** [review]

File: pkg/runtime/ast/truncate.go (hunk 1 above)
Comment (dongzl) [reply pkg/runtime/ast/truncate.go]: Thanks, fix.

File: pkg/runtime/plan/truncate.go (hunk 2 above)
Comment (dongzl) [reply pkg/runtime/plan/truncate.go]: Thanks, fix.

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**